src/clients/node/src/test.ts

```diff
@@ -562,20 +562,24 @@ test('can get account transfers', async (): Promise<void> => {
 
 async function main () {
   const start = new Date().getTime()
+  var fail_count = 0
+  
   try {
     for (let i = 0; i < tests.length; i++) {
         await tests[i].fn().then(() => {
         console.log(tests[i].name + ": PASSED")
       }).catch(error => {
         console.log(tests[i].name + ": FAILED")
-        throw error
+        fail_count += 1;
       })
     }
-    const end = new Date().getTime()
-    console.log('Time taken (s):', (end - start)/1000)
   } finally {
     await client.destroy()
   }
+
+  const end = new Date().getTime()
+  console.log('Time taken (s):', (end - start)/1000)  
+  process.exit(fail_count);
 }
 
 main().catch((error: AssertionError) => {
```

Comment:
Let’s rather exit process in the .catch to main here, to have a guarantee that any exception results in abnormal termination